### PR TITLE
Restructure site

### PR DESCRIPTION
**Pull request by jimmac**
Provide contact info, separate installation and usage page, link to articles, talks and blog posts, provide a logo/presskit.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -16,108 +16,6 @@ However, this tool doesn't *require* using an OSTree based system. It works equa
...
-
-```Dockerfile
-LABEL com.github.containers.toolbox="true"
-```
-
-The label `com.github.debarshiray.toolbox="true"` was used in previous versions of toolbx but is currently deprecated.
-
-[![Zuul](https://zuul-ci.org/gated.svg)](https://softwarefactory-project.io/zuul/t/local/builds?project=containers/toolbox) [![Daily Pipeline](https://softwarefactory-project.io/zuul/api/tenant/local/badge?project=containers/toolbox&pipeline=periodic)](https://softwarefactory-project.io/zuul/t/local/builds?project=containers%2Ftoolbox&pipeline=periodic) [![Arch Linux package](https://img.shields.io/archlinux/v/community/x86_64/toolbox)](https://www.archlinux.org/packages/community/x86_64/toolbox/) [![Fedora package](https://img.shields.io/fedora/v/toolbox/rawhide)](https://src.fedoraproject.org/rpms/toolbox/)
\ No newline at end of file
+See our [detailed guide](install).
```
Comment (debarshiray) [inline index.md]: Cool, I like this!

Maybe we should do the same to the `README.md` of the [code repository](https://github.com/containers/toolbox) too? Or not? What do you think?

File: index.md [hunk 2]
```diff
@@ -34,6 +34,10 @@ See our [detailed guide](install).
 * [Toolbox -- a fall 2019 update](https://debarshiray.wordpress.com/2019/11/01/toolbox-a-fall-2019-update/)
 * [Video talk -- Toolbox: using Silverblue for development](https://www.youtube.com/watch?v=BGXs0W6NRBM) by Debarshi Ray
 
+## Press Information
+We are very happy to answer questions from journalists and tech writers. The toolbox logo is licensed CCBYSA3.0 and can be [downloaded here](/assets/logo/toolbx-logo.zip).
+
+Press queries, including requests for comments and interviews can be directed to [press@containertoolbx.org](mailto:press@containertoolbx.org).
```
Comment (debarshiray) [inline index.md:40]: Do we need to create a `press@containertoolbx.org` email alias or mailing list? How is it handled for the other projects?

The Podman folks might have something set up already. We could ask them to let us piggy back on their thing.

What do you think?

**Review by debarshiray — COMMENTED** [review]
Thanks, @jimmac

File: index.md (hunk 2 above)
Comment (jimmac) [reply index.md:40]: Oh yea, we went with a placeholder live :) I was gonna have you replace it with what you got.

File: index.md (hunk 1 above)
Comment (jimmac) [reply index.md]: I'd have the README in the website repo relate to the website rather than toolbx.

Comment (debarshiray) [reply index.md]: I meant, keeping the `README.md` of the [code repository](https://github.com/containers/toolbox) loosely synchronized with the `index.md` of the website.

Comment (jimmac) [reply index.md]: Alternatively link to it instead of `install.md`

File: index.md (hunk 2 above)
Comment (debarshiray) [reply index.md:40]: We got `press@lists.podman.io` created and updated the website a few weeks back.

**Pull request merged**